src/components/views/elements/AppTile.js

```diff
@@ -172,18 +172,32 @@ export default React.createClass({
     */
     _onDeleteClick: function() {
         if (this._canUserModify()) {
-            console.log("Delete widget %s", this.props.id);
-            this.setState({deleting: true});
-            MatrixClientPeg.get().sendStateEvent(
-                this.props.room.roomId,
-                'im.vector.modular.widgets',
-                {}, // empty content
-                this.props.id,
-            ).then(() => {
-                console.log('Deleted widget');
-            }, (e) => {
-                console.error('Failed to delete widget', e);
-                this.setState({deleting: false});
+            // Show delete confirmation dialog
+            const QuestionDialog = sdk.getComponent("dialogs.QuestionDialog");
+            Modal.createTrackedDialog('Delete Widget', '', QuestionDialog, {
+                title: _t("Delete Widget"),
+                description:
```

Comment:
I think the description can be a string

Reply:
I'm confused by this comment. I'm not sure what change you are suggesting.

This is very similar to usage in places like - https://github.com/matrix-org/matrix-react-sdk/blob/master/src/components/structures/UserSettings.js#L427-L433

Reply:
You could just pass the output of _t rather than wrap it in a div

Reply:
I see. Thanks for the clarification @t3chguy. I've removed the div, as suggested ... but it seems a bit inconsistent with the usage in lots of the rest of the code ...

```
components/views/login/RegistrationForm.js-                    description:
components/views/login/RegistrationForm.js-                        <div>
--
components/views/messages/TextualBody.js-                description:
components/views/messages/TextualBody.js-                    <div>
--
components/views/rooms/MemberInfo.js-                    description:
components/views/rooms/MemberInfo.js-                        <div>
--
components/views/rooms/MessageComposer.js-            description: (
components/views/rooms/MessageComposer.js-                <div>
--
components/views/rooms/RoomSettings.js-            description:
components/views/rooms/RoomSettings.js-                <div>
--
components/views/rooms/RoomSettings.js-            description: (
components/views/rooms/RoomSettings.js-                <div>
--
components/views/settings/ChangePassword.js-            description:
components/views/settings/ChangePassword.js-                <div>
...
```